bin/verify-clis.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Verify existence of gh and docker CLIs
+# https://stackoverflow.com/a/677212
+set -e
+if ! command -v docker &> /dev/null
+then
+    echo "docker CLI could not be found"
+    exit 1
+fi
+if ! command -v gh &> /dev/null
```

Comment:
What are we using `gh` for?

Reply:
we use it to create the draft GitHub release:
https://github.com/readmeio/rdme/blob/3b153c84159182499e7dd66fcc276cfcb05e1bfd/.releaserc.yml#L42